app/controllers/reviews_controller.rb

```diff
@@ -9,17 +9,19 @@ def show
   end
 
   def new
-    @review = Review.new
+    @medium = Medium.find(params[:medium_id])
```

Comment:
You could DRY up here by allowing your :set_medium before_action to run on new and create